docker-images/opentelemetry-collector/builder.template.yaml

```
@@ -36,3 +36,4 @@ processors:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/probabilisticsamplerprocessor v$OTEL_COLLECTOR_VERSION
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/tailsamplingprocessor v$OTEL_COLLECTOR_VERSION
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v$OTEL_COLLECTOR_VERSION
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor v$OTEL_COLLECTOR_VERSION
```

```suggestion
  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor v$OTEL_COLLECTOR_VERSION
```

Note the pattern above, it's not the github URL but the `go.mod` import path